### PR TITLE
First version with new parameters to execute commands and create wind…

### PR DESCRIPTION
**Pull request by jmalvarezf-lmes**
Hi,

Here at Leroy Merlin Spain we need to compile some solutions that are Microsoft-Only and the only way we found it possible to accomplish the build with Google Cloud Build was using the windows remote cloud-build-step. As we found some difficulties for our deployment (network and regions basically) and we found that the method to copy workspace was very slow, we customized the windows-builder with success and we wanted to share the changes in case they are found useful.

Basically we added the following parameters:

- network
- bubnetwork
- region
- zone
- not-copy-workspace

We also made a very small change to allow the builder to get the password from an env var.

Hope it is useful. Feel free to ask for any change.

Thanks!

### CODE REVIEW

**Comment by jmalvarezf-lmes** [comment]
@googlebot I signed it!
